src/commands/createStaticWebApp/OutputLocationStep.ts

```diff
@@ -16,7 +16,7 @@ export class OutputLocationStep extends AzureWizardPromptStep<IStaticWebAppWizar
         const defaultLocation: string = 'build';
         wizardContext.outputLocation = (await ext.ui.showInputBox({
             value: getWorkspaceSetting(outputSubpathSetting, wizardContext.fsPath) || getWorkspaceSetting(appArtifactSubpathSetting, wizardContext.fsPath) || defaultLocation,
-            prompt: localize('publishLocation', "Enter the path of your build output relative to your app's location. For example, setting a value of 'build' when your app location is set to 'app' will cause the content at 'app/build' to be served.")
+            prompt: localize('publishLocation', "Enter the path of your build output relative to your app's location. For example, setting a value of 'build' when your app location is set to 'app' will cause the content at 'app/build' to be served. Leave blank to use root.")
```

Comment:
> Leave blank to use root.

I don't think this is super accurate. Sounds like it'll use the app's location, not the root of the workspace. Also, I think Anthony's comment was more for projects that don't have a build (aka they're not using a framework), so this step doesn't matter.

What about this:

> Enter the path of your build output relative to your app's location, or leave blank if it has no build. For example, setting a value of 'build' when your app location is set to 'app' will cause the content at 'app/build' to be served.

Reply:
> Also, I think Anthony's comment was more for projects that don't have a build (aka they're not using a framework), so this step doesn't matter.

https://docs.microsoft.com/en-us/azure/static-web-apps/front-end-frameworks

I don't think that this is accurate either.  I'm pretty sure all Static Web Apps need to have a build output location otherwise we would let users just the skip the step.  Of course there are cases where the build script outputs all the code in the app folder itself, in which case you'd just leave it blank to refer to the app folder.

Reply:
Actually, I take that back, it _is_ an optional parameter.

Reply:
You're welcome to suggest a better wording 😊 I was kind of going off the "No Framework" tab in this tutorial


https://docs.microsoft.com/en-us/azure/static-web-apps/getting-started?tabs=vanilla-javascript
![Screen Shot 2021-02-18 at 2 33 58 PM](https://user-images.githubusercontent.com/11282622/108432938-0388ed00-71fa-11eb-93ff-916d6d45cc6d.png)

Reply:
Nuts. Eric wins again.  This is why you don't challenge the boss.